app/javascript/react/containers/BeersContainer.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import BeerTile from '../components/BeerTile'
+
+
+class BeersContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      beerList: []
+    }
+  }
+
+  componentDidMount() {
+    fetch("/api/v1/beers")
+    .then(response => {
+      if (response.ok) {
+        return response;
+      }
+      else {
```

Comment:
Proper js style is to have `} else {` all on one line, rather than breaking it up like this